packages/fxa-auth-server/lib/senders/emails/templates/welcome.ts

```diff
@@ -0,0 +1,17 @@
+export const render = () => {
+  return `
+    <mjml>
+      <mj-body>
+      <mj-include path="./lib/senders/emails/css/style.css" type="css" css-inline="inline" />
+        <mj-section>
+          <mj-column>
+            <mj-image width="32" height="32" src="https://accounts-static.cdn.mozilla.net/images/16821f55.firefox-logo.svg"></mj-image>
+            <mj-divider></mj-divider>
+            <mj-text css-class="large-text">Welcome, Yogita! Thank you for signing up </mj-text>
+            <mj-text><h1 data-l10n-id="hello-world">Is this localized</h1></mj-text>
```

Comment:
Upon running `yarn write-emails` in the `auth-server` root directory, `welcome.html` and `welcome.text` will be generated having localized values as setup in the corresponding `.ftl` files